app/src/ui/components/Card.module.css

```diff
@@ -1,9 +1,10 @@
 .card {
+  composes: color-scheme-secondary from global;
   width: 276px;
-  flex-direction: column;
+
   border-radius: 10px;
-  border: 1px solid var(--second-color);
-  background: var(--second-color);
+  border: 1px solid;
+
   display: flex;
```

Comment:
can be deleted also cause container is already flex

Reply:
also

`.buttons-box {
  flex-direction: column;
  gap: 10px;
  width: 100%;
}`

remove flex-direction here